codespan/Cargo.toml

```diff
@@ -11,10 +11,11 @@ documentation = "https://docs.rs/codespan"
 edition = "2018"
 
 [dependencies]
+codespan-reporting = { path = "../codespan-reporting", version = "0.8.0" } # CODESPAN
```

Comment:
Hmm ... Is there a way to make this conditional on a feature, so that I don't have to depend on codespan-reporting unless I use it? A default-on feature would be fine.

Reply:
Yes, we could do this!

Reply:
Done in 77160d0 :)